package.json

```diff
@@ -45,7 +45,7 @@
     },
     "advancedLocalNodeModules": {
```

Comment:
This is going to be messy, but it's better for future code and better for users:

Let's remove this current config and create a new one ("overrideNodeModulesPaths"?).

This new setting will be an array type, with a schema something like this:
```json
"overrideNodeModulesPaths": {
  "title": "Override the path to the local node_modules folder",
  "description": "Multiple override paths can be separated with a comma.",
  "type": "array",
  "default": [],
  "items": {
    "type": "string"
  }
}
```

Then in the `activate()` section you'll need to migrate current users of the setting over to this new config, similar to [this](https://github.com/AtomLinter/linter-pylint/blob/71463851dc3a3fec39c4cdebbfa6b9c09b25170f/lib/main.js#L67).

The reason I'm saying this should be done is you are currently depending on users being _exact_ in following your directions `foo/bar,foobar`. But as we all know users are the absolute worst at following directions and will do things like `foo/bar, foobar`, giving your current code a nice extra space in there. The array config type offloads all the extra massaging of the string to Atom and lets us just focus on using the values.